polaris-core/src/main/java/org/apache/polaris/core/secrets/SecretReference.java

```diff
@@ -51,7 +51,7 @@
  * the stored "secret material" as well as the referencePayload and any associated keys used for
  * encryption.
  */
-public class UserSecretReference {
+public class SecretReference {
```

Comment:
nit: Is this rename critical to adding SigV4 support? It would have been nicer to do it in a separate PR without functional changes (to reduce distraction) :slightly_smiling_face: